src/main.rs

```diff
@@ -675,30 +538,139 @@ async fn main() {
             .unwrap_or(172800),
         beaconchain_api_key: env::var("BEACONCHAIN_API_KEY").unwrap_or("".to_owned()),
     });
+    tracing::info!("Config: {shared_state:?}");
 
     let app = Router::new()
         .route("/", get(alive))
-        .route("/info", get(info))
-        .route("/eth/proof/:block_hash", get(get_eth_proof))
-        .route("/eth/head", get(get_eth_head))
-        .route("/avl/head", get(get_avl_head))
-        .route("/avl/proof/:block_hash/:message_id", get(get_avl_proof))
-        .route("/beacon/slot/:slot_number", get(get_beacon_slot))
+        .route("/versions", get(versions))
+        .route("/v1/info", get(info))
+        .route("/v1/eth/proof/:block_hash", get(get_eth_proof))
+        .route("/v1/eth/head", get(get_eth_head))
+        .route("/v1/avl/head", get(get_avl_head))
+        .route("/v1/avl/proof/:block_hash/:message_id", get(get_avl_proof))
```

Comment:
Changing URLs will have an impact on our monitors as well.
We should run the old and new bridge API in parallel until the migration is completed.

Reply:
@luka-ethernal yes but I think @0xSasaPrsic is saying something else, it's that we also need non-`v1` endpoints for a deprecation period before we fully switch over. I agree as well, we need both for now.

Reply:
Yes, I see. I was thinking we add versioning now and skip the deprecation phase since we have no external users yet (do we?). But if needed, I can do it.

Reply:
Tagging @vthunder to help with a suggestion.

Reply:
We use it in almost all our validium integrations, all of which are internal but until we change all of them (in a month maybe?), good to retain.

Reply:
Ok, then I'll maintain the old endpoints till we can phase them out. I will need to change the docs as well and bring back the beacon endpoint.